.github/ISSUE_TEMPLATE.md

```diff
@@ -21,8 +24,7 @@ Place an `x` between the square brackets on the lines below for every satisified
 ### Steps to Reproduce
 
 <!--
-Please add a series of steps to reproduce the problem. See https://stackoverflow.com/help/mcve for in depth information
-on how to create a mimimal, complete, and verifiable example.
+Please add a series of steps to reproduce the problem. See https://stackoverflow.com/help/mcve for in depth information on how to create a mimimal, complete and verifiable example. TL;DR version: *Ideally*, include *all* (but *only*) the necessary *code, configuration and commands* that we can copy-paste into an empty project and run to see the issue. Not every issue has to meet that ideal, but for those that can, it ensures we have all the info needed to look for a solution to the problem -- and you may even solve it yourself while narrowing the code down!
```

Comment:
can we just copy what SO has written and provide the URL? ..

* * *

Please provide a [Minimal, Complete, Verifiable Example](https://stackoverflow.com/help/mcve).

- **Minimal** – Use as little code as possible that still produces the same problem
- **Complete** – Provide all parts needed to reproduce the problem
- **Verifiable** – Test the code you're about to provide to make sure it reproduces the problem